tests/file_storage/tests.py

```diff
@@ -944,6 +944,18 @@ def test_filefield_default(self):
         self.assertEqual(obj.default.read(), b"default content")
         obj.default.close()
 
+    def test_filefield_db_default(self):
+        temp_storage.save("tests/db_default.txt", ContentFile("default content"))
+        obj = Storage.objects.create()
+        self.assertEqual(obj.db_default.name, "tests/db_default.txt")
+        self.assertEqual(obj.db_default.read(), b"default content")
+        obj.db_default.close()
```

Comment:
Do we need this call to be a cleanup call?
```suggestion
        self.addCleanup(obj.db_default.close)
```

Reply:
This caused some failures here: https://github.com/django/django/actions/runs/10251452778/job/28359533468
I might have been doing it wrong but reverting for now